rebar.config

```diff
@@ -16,8 +16,8 @@
 {profiles, [
     {local, []},
     {test, [
-        {deps, [{aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"42cd47d"}}},
-                {aesophia_cli, {git, "git://github.com/aeternity/aesophia_cli", {tag, "v4.3.1"}}}]},
+        {deps, [{aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"98a4049"}}},
+                {aesophia_cli, {git, "git://github.com/aeternity/aesophia_cli", {ref, "45862a3"}}}]},
```

Comment:
This commit is from May 26. Why not a newer one?

Reply:
Good question. Will take another look

Reply:
Now at absolute latest